Exadel.Compreface/Exceptions/ServiceException.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Exadel.Compreface.Exceptions
+{
+    public class ServiceException : Exception
+    {
+        public ServiceException(string _methodName, string _exceptionMessage) : base(ServiceExceptionMethod(_methodName, _exceptionMessage))
```

Comment:
there is a standart way of creating custom user-defined exceptions, just follow this link 
https://learn.microsoft.com/en-us/dotnet/standard/exceptions/how-to-create-localized-exception-messages

Reply:
yeah, I know about it. But if we will use this custom example, we will need to write a lot of identical strings.